website/static/js/registrationModal.js

```diff
@@ -52,14 +52,15 @@ var RegistrationViewModel = function(confirm, prompts, validator) {
 
     var validation = [{
         validator: function() {
-            var endEmbargoDateTimestamp = self.embargoEndDate().getTime();
+            var timeZoneOffset = self.embargoEndDate().getTimezoneOffset() * (60 * 1000);
```

Comment:
We should still be using moment.js in the site. There's probably a better way to do this using moment. The things I want to avoid are 1) using magic numbers; and 2) anything by hand related to time because time is the *worst*.